### ISSUE TITLE
cause unicode problem

### ISSUE THREAD

**Issue by webszy** [alloc/vite-plugin-compress#10]
i use the vite2.6.13 and vite-plugin-vue2,1.9.0,the files parsed unicode problem,and cannot correctly display
then output all the file just like this:
�4�ŒÔV7„‰«î§î§•œõž@º�¿ÕaÑ9å´�y�K+��«†ÎÙ��¡ben÷.µÓÆ§(&Žò�ªzU(�� Ñ�íP1gÛŽž� [rJ��Vð’��ƒk]/ó��r(ˆŸ!¢ÈJQì•ÿo#Áô�0ßUM5¾û™9�wU¬˜éØ*‰7”#!Z²�¹™�»I��^£Ô¤$¨yc�ëH¶6œ¤y7Û�¢eæ�«êj�:÷ëåö¹obSH;W{O£Õ…�P;bAHGQb]€�0—µ k45�X…!Å_�ihTQ¥0�’Í�õ&�'Y�+\°�L°Ú É¾-1ðþ/NGõ8’|ú±×zøEÙ2MÒ²Œfe��ßO~�¤¹Ó»Ç‹é&œgp� °'GÌ†g�´Æë�_;#9º�.Æìº÷4�­F�£w®(×qh<ãøƒ�/�¼©}°/îým˜m‡ruoëH �"ƒ;”¥ªŠ0V�==hïC€’‘êPc…é½L�­IÖôé1„®«RÛ#©�9�PÕ©�žc�ÊÀê-Ðºj€ø‘dãqÕ+ÆÏÇ4 ‹ÔÂ�PŠÄ²§cvÄM¾È�æô3×4p$Zëœp&^,{†±VÖT24oW?e=œú¸^ïh{¡='Eë)�¯;ãDõÈê•yéö…Ø{ám*¶�Ø3–aq­ZÇ…���}«hš®¶å‚Ù1Hi¶X:•í=³ÂgT�¥Þe‹�ÿ= c�“Þ�ÖÈ�¸®�Û»ÃbÉž…Ç�,Û,`��HøÛl‚xA3º�†�‡¤�Ÿ'|í7C:k�KŒñÈ&ˆ�t�Aë�÷�½+Z «CÅæ�ßï€±ëx,Y¶?â�e�¡ôš�òn ·±�IZá|µ,û8¼ò�! �7.D�Ge€BÍ«Ö†ÝªÓñt3�ÇÄ¦û—��§=”�Q[.YÓtRÏÆuôïà�Àn�¯îa§²Ÿ

**Comment by klakesen** [comment]
I have met the same problem

**Comment by tianjie326git** [comment]
me too

**Comment by DemonGao** [comment]
me too

**Comment by andrecasal** [comment]
Me too.

Any idea how to fix this?

**Comment by andrecasal** [comment]
Yeah, the problem is having no support for Gzip. Use https://github.com/vbenjs/vite-plugin-compression instead.

**Comment by aleclarson** [comment]
If you want this fixed, fork this repo and implement a minimal reproduction in its `./demo` folder. Thank you

**Comment by andreacfromtheapp** [comment]
fwiw, same here. repo: github.com/gacallea/anerandros.info/

```
    "vite": "^2.9.7",
    "vite-plugin-compress": "^2.1.1"
```

```
import { defineConfig } from 'vite'
import compress from 'vite-plugin-compress'

export default defineConfig({
  plugins: [compress()],
})
```

**Comment by Makio64** [comment]
Got the same error, just installing compress and adding it to my vite project.

**Comment by bestwestern** [comment]
I get the same problem.
Windows 10
node 16.17
vite-plugin-compress 2.1.1
vite 3.0.7
